feature/zero-dollar-auth (#2302)
mercadopago/devsite-docs: guides/zero-dollar-auth/integration-test.pt.md

# Teste de integração

> Para probar la validación de Zero Dollar Auth, siga las siguientes instrucciones.

Reviewer: Está en ES, Gosti, y a la inversa para el archivo en ES

> Para probar la validación de Zero Dollar Auth, siga las siguientes instrucciones.

Author: ```suggestion
Para testar a validação de Zero Dollar Auth, siga as seguintes instruções.
```

> # Validação de integração

Reviewer 2: ```suggestion
# Teste de integração
```

Reviewer 2: @abderossi - Gosti, estou achando essa seção um pouco esquisita. Ao ler, não consigo ver um passo a passo definido para testar/validar a integração. Vou deixar alguns exempolos:

_- Adicionar cabeçalho "X-Test-Token: true": Para todos os curls de token de cartão, é importante adicionar o cabeçalho X-Test-Token: true. Isso indicará à API que ela deve processar o consumo de ZDA no ambiente de testes (Sandbox)._ - Mas quais são esses curls? Como que o usuário vai adicionar o header? Qual o endpoint de token de cartão? Ele adiciona o cabeçalho nesse curl e qual a resposta esperada? O que ele deverá receber neste request que conversa com as etapas seguintes de teste? Sem esses detalhes, a seção fica vazia e pouco intuitiva.

_Simular aprovações e rejeições nos testes - Para simular aprovações, utilize o cartão específico 4074090000000004 no curl do endpoint [/v1/payments](https://docs02.mercadopago.com.ar/developers/pt/docs/zero-dollar-auth/integration). Para simular rejeições, use qualquer outro cartão de teste._ - Mesma coisa aqui. Cadê o `curl` de payments? Eu devo adicionar o número de cartão no `curl`, certo? Mas aonde? No body da requisição? Eu devo inseri-lo em qual parâmetro? Este campo tem alguma limitação? Tem algum exemplo que possa guiar o usuário nesta etapa?

_Realizar transações com dados do cartão abertos - Para poder realizar transações com os dados do cartão abertos, diretamente na chamada da API, é necessário que o vendedor seja [PCI Compliant](https://docs02.mercadopago.com.ar/developers/pt/docs/security/pci). Caso contrário, esses dados não podem ser transacionados no back-end da sua aplicação._ - Gosti, o que seria "dados do cartão abertos"? Temos algum contexto disso? No texto diz "Para poder realizar transações com os dados do cartão abertos, diretamente na chamada da API", mas qual API? Para que serve esta etapa afinal?

De maneira geral, Gosti, acho que essa seção precisa ser totalmente reescrita. Do ponto de vista do usuário, ela não ensina efetivamente a testar a integração.

O time de ZDA precisa passar todas essas informações para termos uma doc funcional e que atenda aos requisitos mínimos necessários para testar.
